code/chart.js

```diff
@@ -1,4 +1,27 @@
 //DOM-selector for the canvas 👇
```

Comment:
Hey Daniel, a bit hard to review Javascript when struggling with the understanding. But I know that you  have that 😄.  You have a good structure in the code, easy to follow with the indents and so on.